std/algorithm/sorting.d

```diff
@@ -640,9 +640,12 @@ if (isRandomAccessRange!Range && hasLength!Range && hasSlicing!Range && hasAssig
             // Loop invariant
             version(unittest)
             {
-                import std.algorithm.searching : all;
-                assert(r[0 .. lo].all!(x => !lt(p, x)));
-                assert(r[hi + 1 .. r.length].all!(x => !lt(x, p)));
+                // this used to import std.algorithm.all, but we want to save
+                // imports when unittests are enabled if possible.
```

Comment:
I added the comment here to prevent regression.

Reply:
StdUnittest was reverted because apparently `-deps` includes _all_ unittests and thus usage of StdUnittest caused breakage for projects using `-deps`